lib/registry.py

```diff
@@ -518,7 +519,7 @@ def manifest_upload(self, manifest):
       self.request("PUT", url, {201}, data=manifest,
                    headers={ "Content-Type": TYPES_MANIFEST["docker2"] })
 
-   def request(self, method, url, statuses={200}, out=None, **kwargs):
+   def request(self, method, url, statuses={200}, out=None, expdigest=None, **kwargs):
```

Comment:
Document the new keyword argument in the docstring.

Also style — looks like this line may be more than 80 characters? If so please wrap.